common/datasets/switchboard/constants.py

```diff
@@ -0,0 +1,17 @@
+SUBDIR_PREFIX = "Switchboard"
+
+durations = {"train": 311.62, "hub5e00": 3.61, "hub5e01": 6.17, "rt03s": 6.27}
+
+concurrent = {
+    "train": 60,
+    "hub5e00": 10,
```

Comment:
maybe 5? so that the size per sub corpus for dev and test are the same?

Reply:
Hmm, if we want to be very accurate we could use 7 and 12...
I am not sure about this...